playbooks/app_dev/builds.adoc

```diff
@@ -411,6 +417,79 @@ The following is a snippet of code demonstrating the use of a BuildConfig
 <1> Specifies the source repository that will be injected into the build
 <2> Allows the Docker socket to be available inside the builder. Essential when the builder is use to build and push new Docker images
 
+== Build Types
+
+Depending on the build strategy, one or more of the following types can be applied:
+
+* Git
+* Dockerfile
+* Binary
+
+
+=== Git
+
+Source code is retrieved from a Git based repository
+
+=== Dockerfile
+
+An inline dockerfile can be specified within the *BuildConfig* object
+
+[source]
+----
+{
+ "source" : {
+    "type" : "Dockerfile",
+    "dockerfile": "FROM centos:7\nRUN yum install -y httpd"
+ },
+}
+
+----
+
+=== Binary
```

Comment:
I believe this is only applicable to newer versions of OpenShift? - if so, maybe add another "tag" to this section indicating the minimum version of OpenShift (if known?)